examples/shouldi/shouldi/cli.py

```diff
@@ -182,12 +183,67 @@ async def is_lang_javascript(self, repo):
                 return {"javascript": True}
 
 
+DATAFLOW_SA_JAVASCRIPT = DataFlow.auto(
+    pypi_package_json,
+    pypi_latest_package_version,
+    pypi_package_url,
+    pypi_package_contents,
+    cleanup_pypi_package,
+    run_npm_audit,
+    GetSingle,
+)
+
+DATAFLOW_SA_JAVASCRIPT.seed.append(
+    Input(
+        value=[run_npm_audit.op.outputs["report"].name,],
+        definition=GetSingle.op.inputs["spec"],
+    )
+)
+
+
+@op(
+    inputs={"repo": clone_git_repo.op.outputs["repo"]},
+    outputs={"result": SA_RESULTS},
+    conditions=[is_lang_javascript.op.outputs["javascript"]],
+)
+async def run_javascript_sa(self, repo):
+    """
+    Run JS static analysis
+    """
+    setup_kwargs = SetupPyKWArg.get_kwargs(
```

Comment:
This is a CLI util that can be also used programaticlly to extract information from setup.py files, but only for Python setup.py files. https://intel.github.io/dffml/master/cli.html#kwarg

You won't need this here